### PR TITLE
feat(card): support filled variant

### PR DESCRIPTION
**Pull request by cszhjh**
## Checklist

List of tasks you have already done and plan to do.

- [ ] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

Support Filled Variant

## Issues

#1773

## Related Links

Links related to this pr.

### CODE REVIEW

File: packages/varlet-ui/src/card/Card.vue [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
   <div
     ref="card"
-    :class="classes(n(), [isRow, n('--layout-row')], [outline, n('--outline')], formatElevation(elevation, 1))"
+    :class="
+      classes(
+        n(),
+        [isRow, n('--layout-row')],
+        [outline, n('--outline')],
+        [filled, n('--filled')],
+        formatElevation(elevation, 1)
```
Comment (haoziqaq) [inline packages/varlet-ui/src/card/Card.vue:10]: The elevation should be disable on filled mode.

File: packages/varlet-ui/src/styles/common.less [hunk 2]
```diff
@@ -97,6 +97,8 @@
   --color-surface-container-highest: hsla(var(--hsl-surface-container-highest), 1);
   --hsl-inverse-surface: 0, 0%, 20%;
   --color-inverse-surface: hsla(var(--hsl-inverse-surface), 1);
+  --hsl-filled: 0, 0%, 93%;
+  --color-filled: hsla(var(--hsl-filled), 1);
```
Comment (haoziqaq) [inline packages/varlet-ui/src/styles/common.less:101]: We don't need filled variable in common. I suggest that it only be in the card component.

**Review by haoziqaq — APPROVED** [review]

**Comment by chouchouji** [comment]
Thanks for your contribution!

File: packages/varlet-ui/src/card/card.less [hunk 3]
```diff
@@ -184,4 +185,8 @@
   &--outline {
     border: thin solid var(--card-outline-color);
   }
+
+  &--filled &__floater {
+    background-color: var(--card-filled-color);
```
Comment (haoziqaq) [inline packages/varlet-ui/src/card/card.less:190]: rename to var(--card-filled-background)

**Review by haoziqaq — APPROVED** [review]

**Pull request merged**